src/SemCor.jl

```diff
@@ -2,6 +2,48 @@ struct SemCor{S}
     filepaths::Vector{S}
 end
 
+"""
+    SemCor()
+
+Creates a SemCor instance for lazy loading the corpus.
+
+# Example Usage
+
+```julia
+julia> corp = load(SemCor())
+Channel{CorpusLoaders.Document{Array{Array{Array{CorpusLoaders.TaggedWord,1},1},1},String}}(sz_max:16,sz_curr:16)
+
+julia> index = Dict{String, Vector{Vector{String}}}()
+Dict{String,Array{Array{String,1},1}} with 0 entries
+
+julia> for sentence in flatten_levels(corp, (!lvls)(SemCor, :sent, :word))
+           multisense_words = filter(x->x isa SenseAnnotatedWord,  sentence)
+           context_words = word.(sentence)
+           for target in multisense_words
+               word_sense = sensekey(target)
+               uses = get!(Vector{Vector{String}}, index, word_sense)
+               push!(uses, context_words)
+           end
+       end
+
+julia> index["abandon%1:07:00::"]
+1-element Array{Array{String,1},1}:
+ String["in", "his", "motherland", ";", "in", "the", "spacious", "hunting_grounds", "of", "``"  …  ",", "color", ",", "dance", ",", "design", ",", "and", "thought", "."]
+
+julia> index["abandon%2:38:00::"]
+2-element Array{Array{String,1},1}:
+ String["Jonathan", "wrote", "grimly", "of", "the", "destruction", "of", "Harpers_Ferry", "before", "they"  …  "First_Brigade", "had", "destroyed", "all", "the", "rolling_stock", "of", "the", "B_+_O_Railroad", "."]
+ String["The", "expense", "of_this", "type", "of", "organization", "in", "religious", "life", ","  …  "in", "congregations", "illumines", "the", "nature", "of", "the", "Protestant", "development", "."]
+
+
+```
+
+SemCor is a classical Sense Annotated Corpus. It Has a structure of documents, paragraphs, sentences, words. The words are either tagged with part of speech, or tagged with full lemma, part of speech and sensekey. We expose the word function to get back the words, and the sensekey function to extract the sense key.
```

Comment:
Reduce the length of the above line. - 

```
SemCor is a classical Sense Annotated Corpus. 
It has a structure of documents, paragraphs, sentences, words. 
The words are either tagged with part of speech, or tagged with full lemma, part of speech and sensekey. 
We expose the word function to get back the words, and the sensekey function to extract the sense key.
```

Also, maybe mention something about various SemCor versions.

Reply:
Thankyou @Ayushk4   for all your time and suggestion. I will Implement the above changes